### PR TITLE
DCOS-8101: Add service state to the subline rather then the health status

### PR DESCRIPTION
**Pull request by Poltergeist**
image

image

image

This changes the sub headline on detail pages to states rather then health status.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @daiyiwhale

**Comment by pierlo-upitup** [comment]
Tests seem to be failing `8 tests failed, 1798 tests passed (1806 total in 178 test suites, run time 106.984s)`

**Comment by Poltergeist** [comment]
Tests have been fixed.

File: src/js/components/__tests__/ServiceDetail-test.js [hunk 1]
```diff
@@ -48,6 +49,7 @@ describe('ServiceDetail', function () {
     this.instance =
       TestUtils.findRenderedComponentWithType(this.wrapper, ServiceDetail);
     this.node = ReactDOM.findDOMNode(this.instance);
+
```
Comment (pierlo-upitup) [inline src/js/components/__tests__/ServiceDetail-test.js]: we could live without this line 😉 😛

**Comment by leemunroe** [comment]
Looks like we're doubling up our words here.  How about `Running (x Tasks)`?

**Comment by pierlo-upitup** [comment]
@leemunroe that only makes sense when `Running` is the overall status.

**Comment by leemunroe** [comment]
@pierlo-upitup What are some examples where it doesn't make sense? And do they map to these?

image

**Comment by pierlo-upitup** [comment]
Mmm good point. I think you're actually right. We should stick to the designs. @Poltergeist can you give this a second pass?

**Comment by Poltergeist** [comment]
image

**Comment by rcorral** [comment]
retest this please

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by rcorral** [comment]
flicts!

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

**Comment by Poltergeist** [comment]
**mesosphere-ci** retest this please

File: src/js/pages/task-details/TaskDetail.js [hunk 2]
```diff
@@ -20,6 +20,10 @@ const METHODS_TO_BIND = [
   'onTaskDirectoryStoreSuccess'
 ];
 
+const StatusMapping = {
+  'Running': 'service-running-state'
+};
```
Comment (kennyt) [inline src/js/pages/task-details/TaskDetail.js]: if we have this here and also inside of `SeriviceInfo`, maybe we should abstract it to a constants

Comment (Poltergeist) [inline src/js/pages/task-details/TaskDetail.js]: Moved this to a constant file

**Comment by pierlo-upitup** [comment]
LGTM! 🚢

**Pull request merged**